### DISCUSSION TITLE
User without MFA

### DISCUSSION THREAD

**Discussion by root9191** [authelia/authelia#4990] [Q&A]
Hi, can i also add a User without a MFA, OTP or something. Just with simple password?

**Comment by james-d-elliott** [comment]
Yes, this is how users  are created by default, then if they want to add 2FA it's up to them. You can also configure the access control section to only require 1FA for sites that traditionally need 2FA for particular users.

**Reply by root9191** [reply]
Nope when i add a new User a Second Factor is always requested from Authelia!?

**Reply by nightah** [reply]
This means one of two things. Either the ACL you created is:
1. Set to a policy of two_factor
2. The user in question doesn't have access based on the subject definition

All users are created with just a username and password. If Authelia is requesting the user to setup 2FA the ACL in question either expects 2FA or we cannot verify that the user has access based on the rule so we request an elevated level of access.
